CONTRIBUTING.md

```diff
@@ -1,195 +1,137 @@
-# Contributing to Ballerina
+---
+layout: ballerina-inner-page
+title: Contributing to Ballerina
+permalink: /contribution-guide
+---
```

Comment:
Do we need this information for .md file in this repo?

Reply:
Removed.